### ISSUE TITLE
OT_ME.export_cash uses mutable string argument that is not SWIG compatible

### ISSUE THREAD

**Issue by weissjeffm** [Open-Transactions/opentxs#530]
```cpp
std::string OT_ME::export_cash(const std::string& NOTARY_ID,
                               const std::string& FROM_NYM_ID,
                               const std::string& INSTRUMENT_DEFINITION_ID,
                               const std::string& TO_NYM_ID,
                               const std::string& STR_INDICES,
                               bool bPasswordProtected,
                               std::string& STR_RETAINED_COPY) const
```
that `std::string& STR_RETAINED_COPY` argument is supposed to be for a return value, but I don't think this works properly with SWIG. (eg python strings are immutable).

If this method really needs to return two things, we'll have to come up with an object and let SWIG proxy it. Or maybe one of the return values can be dropped, I'm not sure what `STR_RETAINED_COPY` even is.

**Comment by FellowTraveler** [comment]
Let's say I have some cash in my local cash purse. Since this is CASH, that means if someone gets a copy of it, they can SPEND it.

Therefore, cash is stored in encrypted form at all times.

Let's say I want to give the cash to someone else. In that case, I have to EXPORT it from my local cash purse. When this happens, the cash is re-encrypted to the recipient's public key (or to a passphrase.)

This means I can no longer read the cash! It's now encrypted to someone else's key.

For this reason, export_cash also returns a version of the same cash that is encrypted to MY key. This way the cash can be placed into the payments outbox. This way I can view it the same as any other sent funds. This is the RETAINED cash that you are asking about.

If the recipient loses his copy of the cash before depositing it, I can still recover it from my outbox.

**Comment by weissjeffm** [comment]
Fixed by #558
